cpp/include/cuspatial_test/geometry_generator.cuh

```diff
@@ -399,8 +401,9 @@ auto generate_multipoint_array(multipoint_generator_parameter<T> params,
                    std::size_t{0},
                    params.num_points_per_multipoints);
 
-  auto engine_x = deterministic_engine(params.num_points());
-  auto engine_y = deterministic_engine(2 * params.num_points());
+  auto golden_ratio = (1 + std::sqrt(T{5})) / 2;
+  auto engine_x     = deterministic_engine(golden_ratio * params.num_points());
+  auto engine_y     = deterministic_engine((1 / golden_ratio) * params.num_points());
```

Comment:
During testing, I observed using `num_points()` and `num_points() * 2` as the seeds produced point positions distributed across two parallel diagonals. Maybe that was an artifact of my point count, but it seems rotating around the golden ratio produces a more even distribution.

Reply:
Perhaps this comment should be in the code?